devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -118,11 +118,11 @@ def updated() {
 
 def getCommandClassVersions() {
 	[
-		0x20: 1,  // Basic
```

Comment:
This method is not used in your code. I think it is good practice for learning, but in final commit it is redundant.

Reply:
It's used in L134